mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+from .base import get_coefs, BaseEstimator, _check_estimator
+from ..utils import warn
+
+
+class ReceptiveField(BaseEstimator):
+    """Fit a receptive field model.
+
+    Parameters
+    ----------
+    lags : array, shape (n_lags,)
+        Time lags to be included in the model. This will be combined
+        with `sfreq` to convert these values into indices in the data.
+        If `sfreq` == 1, then these should be indices. If not, then
+        time lags should be in seconds.
+    feature_names : array, shape (n_features,)
+        Names for the input features to the model.
```

Comment:
it'll actually do this automatically if you don't supply `feature_names` (it will set `feature_names` after you call `fit` once it figures out how many features there are). I need to update the docs to say that... thanks for reminding me :)